src/Init/Lean/Parser/Term.lean

```diff
@@ -98,7 +98,6 @@ withPosition $ fun pos =>
 @[builtinTermParser] def borrowed   := parser! symbol "@&" appPrec >> termParser (appPrec - 1)
 @[builtinTermParser] def quotedName := parser! nameLit
 -- NOTE: syntax quotations are defined in Init.Lean.Parser.Command
-@[builtinTermParser] def antiquot   := (mkAntiquot "term" none true : Parser)
```

Comment:
Can we also remove the following definition at `Command.lean`?
```lean
@[builtinCommandParser] def Command.antiquot := (mkAntiquot "command" none true : Parser)
```

Reply:
Yes, good catch! Done.